tests/linkcommand.js

```diff
@@ -324,25 +332,33 @@ describe( 'LinkCommand', () => {
 
 				expect( getData( model ) ).to.equal( 'foo[<$text linkHref="url">url</$text>]bar' );
 			} );
+
+			it( 'should add additional attributes to link if those are falsy but decorator\'s defaultValue is set to true', () => {
```

Comment:
Why storing the `false` value in the model? Also, how do you actually remove the value from the model?

Reply:
![](https://user-images.githubusercontent.com/156149/78411326-98e5b780-760f-11ea-8ac4-4f853a5c42b4.png)

I checked that I correctly read the code – it indeed puts the false value in the model.